### DISCUSSION TITLE
Custom domain can not login

### DISCUSSION THREAD

**Discussion by LuciusChen** [heyform/heyform#48] [Self hosting]
[Nest] 45   - 04/11/2024, 2:08:38 AM   [MongooseModule] usermodels findOne {"email":"xxxxxxxxx@163.com"} {"projection":{}}

**Comment by iMuFeng** [comment]
Can you paste your docker-compose.yml here?

**Reply by LuciusChen** [reply]
deploy with railway

**Reply by LuciusChen** [reply]
![Untitled](https://github.com/heyform/heyform/assets/4947981/1cbe2469-c63e-4c69-ba45-de00a7cd1fa7)

**Reply by iMuFeng** [reply]
Change `APP_HOMEPAGE_URL` to your own domain like this below:

![image](https://github.com/heyform/heyform/assets/1407009/39b31466-24f8-4327-9bb2-bd8760b74ee7)

**Reply by LuciusChen** [reply]
![image](https://github.com/heyform/heyform/assets/4947981/f1d2274d-d402-4a1f-953b-2e47b3918915)

**Reply by LuciusChen** [reply]
I have not modified this variable, it is currently a custom domain name, but still unable to log in.

**Reply by iMuFeng** [reply]
Once you modify the environment variables, you need to redeploy.

**Reply by allankz** [reply]
The APP_HOMEPAGE_URL must be equal the one that you are using on your browser. If you are trying http://localhost but the variable is set to http://127.0.0.1, it will fail.
